tools/modules/software/module_mysql.sh

```
@@ -0,0 +1,110 @@
+	["module_mysql,port"]="3306"
+	["module_mysql,arch"]="x86-64 arm64"
+)
+#
+# Module mysql
+#
+function module_mysql () {
+	local title="mysql"
+	local condition=$(which "$title" 2>/dev/null)
+
+	if pkg_installed docker-ce; then
+		local container=$(docker container ls -a | mawk '/mysql?( |$)/{print $1}')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)